meteor/server/api/ingest/rundownInput.ts

```diff
@@ -1743,6 +1743,14 @@ export function isUpdateAllowed(
 						allowed = false
 					}
 				})
+				if (!allowed) {
```

Comment:
Not sure about the value of this. Is repeated in the `printChanges(partChanges)` below. Although adding the external ID in print changes would be helpful?

Reply:
Ah, yeah that's true. I didn't scroll that far down.. This is unnecessary then.

Reply:
Also no harm in this.